packages/ui-extensions-server-kit/package.json

```diff
@@ -1,19 +1,32 @@
 {
   "name": "@shopify/ui-extensions-server-kit",
-  "version": "0.1.0-alpha.5",
-  "main": "dist/index",
-  "types": "dist/index.d.ts",
+  "version": "0.1.0-alpha.9",
+  "main": "index.js",
+  "module": "index.mjs",
+  "types": "index.d.ts",
   "files": [
     "**"
   ],
+  "exports": {
+    "./package.json": "./package.json",
+    ".": {
+      "require": "./index.js",
+      "import": "./index.mjs",
+      "types": "./index.d.ts"
+    },
+    "./testing": {
+      "require": "./testing.js",
+      "import": "./testing.mjs",
+      "types": "./testing.d.ts"
+    }
```

Comment:
By adding these files, the import syntax of the library is cleaner and more consistent. We can now import:
```ts
import {...} from '@shopify/ui-extensions-server-kit'; // code
import {...} from '@shopify/ui-extensions-server-kit/testing'; // test utils
```